swagger.yaml

```diff
@@ -1682,6 +1828,63 @@ components:
         - START_TO_START
         - FINISH_TO_FINISH
 
+    Notification:
+      type: object
+      description: Notification object that is created upon an event and sent to multiple users
+      required:
+        - message
+        - type
+      properties:
+        id:
+          type: string
+          format: uuid
+          readOnly: true
+        message:
+          description: notification message
+          type: string
+        type:
+          $ref: '#/components/schemas/NotificationType'
+        taskId:
+          description: ID of the subject entity of the notification if applicable, depends on the notification type
+          type: string
+          format: uuid
+        metadata:
+          $ref: '#/components/schemas/Metadata'
+
+    NotificationEvent:
+      type: object
+      description: Received notification by a user
+      required:
+        - notificationId
+        - receiverId
+        - read
+      properties:
+        id:
+          type: string
+          format: uuid
+          readOnly: true
+        notification:
+          description: Full notification object
+          $ref: '#/components/schemas/Notification'
+        receiverId:
+          description: User ID that the notification was sent to
+          type: string
+          format: uuid
+        read:
+          description: Whether the notification has been read
+          type: boolean
```

Comment:
We have a read parameter but how do we handle "dismissing" NotificationEvents i.e. if user presses the X (cross) button? This action should probably remove NotificationEvent from this user by either deleting or setting another status like "dismissed"

Reply:
Apparently I forgot the delete endpoint :D Added it now